.php_cs.dist

```diff
@@ -24,56 +24,10 @@ if (PHP_VERSION_ID < 70000) {
 return PhpCsFixer\Config::create()
     ->setRiskyAllowed(true)
     ->setRules([
-        '@PHP56Migration' => true,
```

Comment:
what about this ?

Reply:
It has no rules: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.14.2/src/RuleSet.php#L289 - maybe `@PHP56Migration:risky` instead?

Reply:
Wow. Yes, please